tests/localosmosis/state_export/scripts/testnetify.py

```diff
@@ -0,0 +1,361 @@
+        help='Validator operator address to replace'
+    )
+
+    parser.add_argument(
+        '--validator-consensus-address',
+        type = str,
+        help='Validator consensus address to replace'
+    )
+
+    parser.add_argument(
+        '--validator-pubkey',
+        type = str,
+        help='Validator pubkey to replace'
+    )
+
+    parser.add_argument(
+        '--account-pubkey',
+        type = str,
+        help='Account pubkey to replace'
+    )
+
+    parser.add_argument(
+        '--account-address',
+        type = str,
+        help='Account address to replace'
+    )
+
+    parser.add_argument(
+        '-v',
+        '--verbose',
+        action='store_true',
+        help='More verbose output'
+    )
+
+    parser.add_argument(
+        '--pretty-output', 
+        action='store_true',
+        help='Properly indent output genesis (increases time and file size)'
+    )
+
+    return parser
+
+def main():
+
+    parser = create_parser()
+    args = parser.parse_args()
+
+    new_validator = Validator(
+        moniker = "val",
+        pubkey = args.validator_pubkey,
+        hex_address = args.validator_hex_address,
+        operator_address = args.validator_operator_address,
+        consensus_address = args.validator_consensus_address
+    )
+
+    old_validator = Validator(
+        moniker = "Sentinel dVPN",
+        pubkey = "b77zCh/VsRgVvfGXuW4dB+Dhg4PrMWWBC5G2K/qFgiU=",
+        hex_address = "16A169951A878247DBE258FDDC71638F6606D156",
+        operator_address = "osmovaloper1cyw4vw20el8e7ez8080md0r8psg25n0cq98a9n",
+        consensus_address = "osmovalcons1z6skn9g6s7py0klztr7acutr3anqd52k9x5p70"
+    )
+
+    new_account = Account(
+        pubkey = args.account_pubkey,
+        address = args.account_address
+    )
+
+    old_account = Account(
+        pubkey = "AqlNb1FM8veQrT4/apR5B3hww8VApc0LTtZnXhq7FqG0",
+        address = "osmo1cyw4vw20el8e7ez8080md0r8psg25n0c6j07j5"
+    )
+
+    print("📝 Opening {}... (it may take a while)".format(args.input_genesis))
+    with open(args.input_genesis, 'r') as f:
+        genesis = json.load(f)
+    
+    # Replace chain-id
+    if args.verbose:
+        print("🔗 Replace chain-id {} with {}".format(genesis['chain_id'], args.chain_id))
+    genesis['chain_id'] = args.chain_id
+
+    # Update gov module
+    if args.verbose:
+        print("🗳️ Update gov module")
+        print("\tSet governance_voting_period to", config["governance_voting_period"])
+    genesis['app_state']['gov']['voting_params']['voting_period'] = config["governance_voting_period"]
+
+    # Update epochs module
+    if args.verbose:
+        print("⌛ Update epochs module")
+        print("\tSet epoch_duration to", config["epoch_duration"])
+        print("\tReset current_epoch_start_time")
+    genesis['app_state']['epochs']['epochs'][0]['duration'] = config["epoch_duration"]
+    genesis['app_state']['epochs']['epochs'][0]['current_epoch_start_time'] = datetime.now().isoformat() + 'Z'

```

Comment:
Can we modify these to show what we are switching from? It just makes things easier when debugging / when dealing with multiple testnets

Reply:
Sure!